addons/account_asset/models/account_asset.py

```diff
@@ -411,7 +411,7 @@ def _compute_entries(self, date, group_entries=False):
     @api.model
     def create(self, vals):
         asset = super(AccountAssetAsset, self.with_context(mail_create_nolog=True)).create(vals)
-        asset.compute_depreciation_board()
+        asset.sudo().compute_depreciation_board()
```

Comment:
Men PLEEEEEEEASE do not do this.

put a SUDO here will mark the validation of the invoices with admin, losing complete traceability :-(

Reply:
The creation of the account.asset.asset is done by the original user.
The depreciation lines will however be done by admin. Is it an issue ?

Reply:
Yes, men, 

On accounting records, we should use ACL to allow write/create things properly, then the change should be in the rules itself and not put sudo into the python methods, because audit the trace of the elements then become a mess, (that's not the case with portal access), but the last write uid being **admin** become such process very difficult.

IF we can avoid do sudo and prepare the proper ACL for the model instead this will help a lot.

Regards and thanks for the quick feedback.

Reply:
Yup, sure thanks for the feedback :)